### PR TITLE
feat: CLS issues - Use explicit image and width on image elements

### PR DESCRIPTION
**Pull request by justyna-13**
# Related issue

Closes #

# Scope of work

# Screenshots of visual changes

# Checklist

- [ ] No commented blocks of code left
- [ ] Run tests and docs
- [ ] Self code-reviewed
- [ ] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue [hunk 1]
```diff
@@ -99,6 +104,14 @@ export default {
...
       default: null,
     },
+    pictureWidth: {
+      type: String,
+      default: null,
+    },
+    pictureHeight: {
+      type: String,
+      default: null,
+    },
```
Comment (AdamPawlinski) [inline packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue:114]: Can you add this props to stories? :)

**Review by AdamPawlinski — COMMENTED** [review]
Good work, only two comments :)

**Review by aniamusial — APPROVED** [review]

**Review by AdamPawlinski — APPROVED** [review]
Great :)

**Review by AdamPawlinski — APPROVED** [review]
No works perfectly well :1st_place_medal:

**Pull request merged**